internal/runtime/runtime_test.go

```diff
@@ -1690,6 +1690,9 @@ func Test_evaluateDynamicVariables(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			rt := &ResourceGroupRuntime{
+				instance: newTestResource(
+					withObject(map[string]interface{}{}),
+				),
```

Comment:
I wonder how is this working when object is empty?

Reply:
It just looks for object to not be nil..it seems like this test does not need object